code/standards/pmdruleset.xml

```diff
@@ -69,7 +69,5 @@
   <rule ref="rulesets/java/unnecessary.xml">
     <exclude name="UselessParentheses" />
   </rule>
-  <rule ref="rulesets/java/unusedcode.xml">
-    <exclude name="UnusedModifier" />
```

Comment:
It doesn't work with the newest version, and to properly use the reports we need to merge some of my other code PRs that fix the underlying issues anyways.